packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -155,6 +158,12 @@ export function createAPI(rpc: RPCProtocol): typeof theia {
         onDidOpenTextDocument(listener, thisArg?, disposables?) {
             return documents.onDidAddDocument(listener, thisArg, disposables);
         },
+        getConfiguration(section?, resource?): theia.WorkspaceConfiguration {
+            return preferenceRegistryExt.getConfiguration(section, resource);
+        },
+        onDidChangeConfiguration(listener, thisArgs?, disposables?): theia.Disposable {
+            return preferenceRegistryExt.onDidChangeConfiguration(listener, thisArgs, disposables);
+        }
```

Comment:
Where is update method?

Reply:
`getConfiguration` returns an object with [`WorkspaceConfiguration`](https://github.com/theia-demo-plugins/wiptheia/pull/30/files/9ab81cffb410c2482f608723458a898f72acd7fe#diff-c12f4cfff29006870b5f6654b73c2f8dR1974) interface. There is method `update` implemented.

Reply:
Ok